src/entry-point.tsx

```diff
@@ -25,6 +25,10 @@ interface RefreshLibraryViewFunc {
     (): void;
 }
 
+interface SearchLibraryItemsFunc {
+    (text: string, callback: Function): void;
```

Comment:
Can we define an interface for the callback to be certain about the function prototype?